microsoftgraph/docs-conceptual/authentication-commands.md

```diff
@@ -97,7 +97,9 @@ To use app-only access, you can load the certificate from either *Cert:\CurrentU
 This type of grant will help when you need interactions in the background without a user to sign in. Support for client secret credentials was added by adding **-ClientSecretCredential** parameter to **Connect-MgGraph**. See [Get-Credential](/powershell/module/microsoft.powershell.security/get-credential) on how to get or create credentials.
 
 ```powershell
-$ClientSecretCredential = Get-Credential -Credential "Client_Id"
+$User = <Application (client) ID>
+$PWord = ConvertTo-SecureString -String <Client Secret> -AsPlainText -Force
+$ClientSecretCredential = New-Object -TypeName System.Management.Automation.PSCredential -ArgumentList $User, $PWord
```

Comment:
Hi @SteveMutungi254 a customer suggested this script in a GitHub issue, stating that the one in our documentation didn’t work. Could you please check if this is correct?
Thank you in advance!